resources/build_resources.sh

```diff
@@ -11,7 +11,7 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 # The path to where the PySide binaries are installed
-PYTHON_BASE="/Applications/Shotgun.app/Contents/Frameworks/Python"
+PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
```

Comment:
Did the PySide executables move in a recent Desktop installation?  The only thing under the Frameworks/Python/bin directory for me was a "python" link back to the Resources folder:
```
66:/Applications/Shotgun.app/Contents/Frameworks/Python/bin ls -l
total 8
lrwxr-xr-x  1 macuser  staff  36 Dec 16 16:11 python@ -> ../../../Resources/Python/bin/python
```

This is for Desktop v1.3.6.